doc/programming_guide/quickstart.rst

```diff
@@ -161,6 +161,16 @@ An example program using keyboard and mouse events is in
 Playing sounds and music
 ------------------------
 
+.. note::
```

Comment:
In my opinion, this is too much to add in the quickstart guide.

Reply:
Agreed. I've cut this down and moved it to the bottom near the footnote.